src/game_inventory.cpp

```diff
@@ -1218,6 +1218,59 @@ item_location game_menus::inv::gun_to_modify( Character &you, const item &gunmod
                          _( "You don't have any guns to modify." ) );
 }
 
+class ereader_inventory_preset : public pickup_inventory_preset
+{
+    public:
+        explicit ereader_inventory_preset( const Character &you ) : pickup_inventory_preset( you ),
+            you( you ) {
+            _collate_entries = true;
+            if( get_option<bool>( "INV_USE_ACTION_NAMES" ) ) {
+                append_cell( [ this ]( const item_location & loc ) {
+                    return string_format( "<color_light_green>%s</color>", get_action_name( *loc ) );
+                }, _( "ACTION" ) );
+            }
+        }
+
+        bool is_shown( const item_location &loc ) const override {
+            return loc->is_ebook_storage();
+        }
```

Comment:
This isn't used anywhere?

Reply:
I "mimicked" some other "inventory_preset" class and in my understanding this part is responsible for marking which inventory items show or do not show on the related menu list.

I can be mistaken and can test locally what happens if I comment this part out.

Reply:
I did test it now - without this part  of the code "E-reader menu" shows incompatible items. Therefore this code is needed.
![image](https://github.com/CleverRaven/Cataclysm-DDA/assets/84157408/c7550697-bb28-471a-9859-8cde040a41c2)